cdp_scrapers/tests/test_prime_gov_utils.py

```diff
@@ -1,60 +1,66 @@
-import enum
 from datetime import datetime
 from typing import List
 
 import pytest
 
-from cdp_scrapers.prime_gov_utils import Meeting, PrimeGovScraper
+from cdp_scrapers.prime_gov_utils import (
+    Meeting,
+    PrimeGovScraper,
+    primegov_strftime,
+    primegov_strptime,
+)
 from cdp_scrapers.scraper_utils import reduced_list
 
 
-class DataItem(enum.IntEnum):
-    ClientId = 0
-    TimeZone = enum.auto()
-    Begin = enum.auto()
-    End = enum.auto()
-    Scraper = enum.auto()
-    NumMeetings = enum.auto()
-    Meetings = enum.auto()
-
-
-test_data = [
-    (
-        "lacity",
-        "America/Los_Angeles",
-        datetime(2022, 9, 1),
-        datetime(2022, 9, 1),
-        PrimeGovScraper(client_id="lacity", timezone="America/Los_Angeles"),
-        2,
-    ),
+begin_dates = [datetime(2022, 9, 1)]
+end_dates = [datetime(2022, 9, 1)]
+scrapers = [PrimeGovScraper(client_id="lacity", timezone="America/Los_Angeles")]
+all_meetings = [
+    list(s.get_meetings(begin_dates[i], end_dates[i])) for i, s in enumerate(scrapers)
 ]
-# append scraped meetings to each test input data set
-test_data = list(
-    map(
-        lambda d: (
-            *d,
-            list(
-                d[DataItem.Scraper].get_meetings(
-                    begin=d[DataItem.Begin], end=d[DataItem.End]
-                )
-            ),
+meeting_counts = [2]
```

Comment:
All the list item indexing to use for the different tests was to messy. Changed to individual lists for the various test input data types. Then in the test, just `zip()` the ones needed to `parametrize()`.

Reply:
I think my only comment for this whole PR is to make the above constants CAPITAL_CASE.